packages/ra-ui-materialui/src/form/Toolbar.tsx

```diff
@@ -51,9 +51,7 @@ import { SaveButton, DeleteButton } from '../button';
  * @prop {string} width Apply to the mobile or desktop classes depending on its value. Pass `xs` to display the mobile version.
  *
  */
-export const Toolbar = <RecordType extends Partial<Record> = Partial<Record>>(
-    props: ToolbarProps<RecordType>
-) => {
+export const Toolbar: FC<ToolbarProps> = props => {
```

Comment:
Fix Toolbar def to avoid errors because of `ToolbarProps` definition change

Reply:
I'd like to keep the toolbar Generic in next

Reply:
ok